deploy/mapping/base.mcf

```diff
@@ -170,6 +170,7 @@ description: C:Provenance->provenance_description
 importUrl: C:Provenance->mcf_url
 importTime: C:Provenance->timestamp_secs
 importDuration: C:Provenance->duration_secs
+isPartOf: C:Provenance->is_part_of
```

Comment:
I think this should be `E:Provenance->E5`, where E5 is:

```
Node: E:Provenance->E5
typeOf: Dataset
dcid: C:Provenance->is_part_of
functionalDeps: dcis
```

Reply:
Sorry I missed this in the other mapping.

Reply:
got it thanks! will update